Tests/UnitTests/Plugins/Cuda/Seeding/SeedfinderCudaTest.cpp

```diff
@@ -6,15 +6,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#include <chrono>
-#include <fstream>
-#include <iomanip>
-#include <iostream>
-#include <sstream>
-#include <utility>
-
-#include <boost/type_erasure/any_cast.hpp>
-
+#include "Acts/Plugins/Cuda/Seeding/Seedfinder.hpp"
```

Comment:
This file is not compiled after this line was moved to the top. Looks like the dependency is broken ?

```
In file included from /home/bomki/Projects/ACTS/Tests/UnitTests/Plugins/Cuda/Seeding/SeedfinderCudaTest.cpp:9:
/home/bomki/Projects/ACTS/Plugins/Cuda/include/Acts/Plugins/Cuda/Seeding/Seedfinder.hpp:30:7: error: ‘Seedfinder’ is not a class template
 class Seedfinder<external_spacepoint_t, Acts::Cuda> {
       ^~~~~~~~~~
```

Reply:
Thanks for reporting! This is exactly what we need the CUDA CI job for.